Resources/Prototypes/Catalog/VendingMachines/Inventories/curadrobe.yml

```
@@ -15,3 +15,5 @@
     ClothingShoesBootsLaceup: 2
     ClothingHeadsetService: 2
     PrinterDocFlatpack: 2 # Corvax-Printer
+    BookPrinterFlatpack: 1 # Cats 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)